public/app/core/components/DynamicImports/SafeDynamicImport.tsx

```diff
@@ -1,27 +1,5 @@
 import React from 'react';
-import Loadable from 'react-loadable';
 
 import { GrafanaRouteComponent } from 'app/core/navigation/types';
 
-import { ErrorLoadingChunk } from './ErrorLoadingChunk';
-import { LoadingChunkPlaceHolder } from './LoadingChunkPlaceHolder';
-
-export const loadComponentHandler = (props: { error: Error; pastDelay: boolean }) => {
-  const { error, pastDelay } = props;
-
-  if (error) {
-    return <ErrorLoadingChunk error={error} />;
-  }
-
-  if (pastDelay) {
-    return <LoadingChunkPlaceHolder />;
-  }
-
-  return null;
-};
-
-export const SafeDynamicImport = (loader: () => Promise<any>): GrafanaRouteComponent =>
-  Loadable({
-    loader: loader,
-    loading: loadComponentHandler,
-  });
+export const SafeDynamicImport = (loader: () => Promise<any>): GrafanaRouteComponent => React.lazy(loader);
```

Comment:
I tried to replace all uses of SafeDynamicImport with React.lazy, it works at runtime, but typescript is complaining for some uses. Typescript is having some issues converting the components our pages return to `GrafanaRouteComponent`, when we have connector(withTheme2(Component))  and other issues when just using the page component has typed queryParams. Think we can address those in a different PR if we want to improve typings.